native/cocos/audio/android/PcmAudioService.cpp

```diff
@@ -113,13 +116,8 @@ bool PcmAudioService::init(AudioMixerController *controller, int numChannels, in
         channelMask,
         SL_BYTEORDER_LITTLEENDIAN};
 
-#if CC_PLATFORM == CC_PLATFORM_ANDROID
-    SLDataLocator_AndroidSimpleBufferQueue locBufQueue = {
-        SL_DATALOCATOR_ANDROIDSIMPLEBUFFERQUEUE,
-        AUDIO_PLAYER_BUFFER_COUNT};
```

Comment:
Why delete these lines?

Reply:
Because openharmony didn't have SL_DATALOCATOR_ANDROIDSIMPLEBUFFERQUEUE before.
But now that it has been added.

Reply:
Got it. 
But you delete the code here, locBufQueue is not defined, right?

Reply:
It is defined below. But I made a mistake here, it should be revert.